### PR TITLE
Update addons.md

### PR DESCRIPTION
**Pull request by NotLewUwU**
Just a new line

### CODE REVIEW

**Review by JT122406 — APPROVED** [review]
@andrei1058 Should be good to merge right in

**Review by OPADILOP — CHANGES_REQUESTED** [review]
You should make the link a little bit shorter. Like - 
https://polymart.org/resource/bedwars1058-adventuremode.3296
This to - 
https://polymart.org/resource/3296/
This it should be good to go

File: docs/BedWars1058/addons.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ If you have issues with the addon, it will probably be better to go to any suppo
 - [TabSorter - Simple and Lightweight Tablist Sorter for BedWars1058.](https://www.spigotmc.org/resources/100842/) - by MrF1yn
 - [ArenaSetup - Setup items for BedWars1058.](https://www.spigotmc.org/resources/97709/) - by Sxhqil
 - [BossBar - A Bossbar Addon for BedWars1058.](https://polymart.org/resource/2881/) - by NotLew_x
+- [AdventureMode - Set your players gamemode to adventure in the waiting lobby.](https://polymart.org/resource/bedwars1058-adventuremode.3296) - by NotLew_x
```
Comment (tomkeuper) [inline docs/BedWars1058/addons.md:56]: Make this line shorter as suggested before after that it can be merged

**Review by tomkeuper — CHANGES_REQUESTED** [review]

**Comment by tomkeuper** [comment]
I just found out that this change is also implented in #57
I will close this in favor of the previous change as it has more in one commit

**Pull request closed without merging**